infra/es_cluster.tf

```diff
@@ -852,6 +869,7 @@ for tar in $todo; do
     ensure_index "$job" "$(index "$job" jobs)"
     ensure_index "$job" "$(index "$job" events)"
     tar --force-local -x -z -f "$(basename "$tar")"
+    patch "$job"
```

Comment:
This step takes a little less than 8 seconds on my machine. The generated sed command is about 70kB in size.